test/extensions/geoip_providers/maxmind/geoip_provider_test.cc

```diff
@@ -32,9 +34,58 @@ class GeoipProviderPeer {
   }
 };
 
-class GeoipProviderTest : public testing::Test {
+namespace {
+const std::string default_city_db_path =
```

Comment:
All these should be constexpr absl::string_view, to avoid runtime init (and would also fix the CI failure).

Reply:
would not just switching from `const` to `constexpr` make those variables compile time init? Some library functions that use this vars in test code expect `std::string`.

Reply:
Nvm, missed that std::string cannot be constexpr

Reply:
The CI failure was about another thing, i had to leave those variables as string as they are being used within TestUtility functions that expect `std::string` inputs.